WebCompiler/Configuration/Settings/SassSettings.cs

```diff
@@ -1,4 +1,5 @@
-﻿using DartSassHost;
+﻿using System.Collections.Generic;
+using DartSassHost;
```

Comment:
Nit: order alphabetically

Reply:
Good spot, I can remove this now as I moved List<string> IgnoreFiles/Folders up to CompilerSettingsCollection.